### PR TITLE
docs: Added ContributionHelp.md

### LINKED ISSUE #1068: Change ContributionHelp file in docs to ContributionHelp.md
1. Change ContributionHelp file in docs to ContributionHelp.md 

2. Add following in main readme file
```
For first time Contributors

First time contributors can read [ContributionHelp.md](docs/ContributionHelp.md) file for help regarding creating issues and sending pull requests.
```

### PR DESCRIPTION
**Pull request by vchrombie**
Fixes #1068 

Changes: [Add here what changes were made in this issue and if possible provide links.]

- Changed ContributionHelp text file to ContributionHelp.md file.

- Styled ContributionHelp.md using markdown syntax.

- Included the ContributionHelp.md link in the README.md.

- Fixed few typos in the README.md.

Screenshots for the change:

### CODE REVIEW

**Comment by vchrombie** [comment]
Can anyone review my PR #1069 please.

**Comment by abishekvashok** [comment]
Please squash your commits.

**Comment by vchrombie** [comment]
Should I close this and open a new PR?

File: README.md [hunk 1]
```diff
@@ -94,14 +94,18 @@ Around 50% of the App is written in [Kotlin](https://kotlinlang.org/). Kotlin is
 Generally, projects are created using package by layer approach where packages are names by layers like `ui`, `activity`, `fragment`, etc but it quickly becomes unscalable in large projects where a large number of unrelated classes are crammed in one layer and it becomes difficult to navigate through them.  
 Instead, we follow package by feature, which at the cost of flatness of our project, provides us packages of isolated functioning related classes which are likely to be a complete self-sufficient component of the application. Each package all related classes of view, presenter, their implementations like Activities and Fragments.  
 A notable exception to this is the `helper` module and data classes like Models and Repositories as they are used in a cross component way.  
-***Note:** The interface contract for Presenter and View is present in `contract` package in each module`*
```
Comment (chiragw15) [inline README.md]: Do not remove asterisk (*) from end. It is used to give overall italic effect. i.e 
```
italic ( bold ( "Note:" ) + "The interface contract for Presenter and View is present in `contract` package in each module" )
```

**Comment by abishekvashok** [comment]
No. Squats commits here itself.

**Comment by chiragw15** [comment]
@vchrombie If you need help in squashing commits http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by vchrombie** [comment]
Sorry to say, I am confused and make my made so many changes to my local repo.
I want to close this PR and I will do the changes again.

**Pull request closed without merging**